### PR TITLE
removes AR-55 from SL vendor

### PR DESCRIPTION
**Pull request by yyzsong**
## About The Pull Request
inb4 xope/ided pr
AR-55 is no longer available roundstart
## Why It's Good For The Game
who gives out 520 points req guns for free
## Changelog
:cl:
del: SLs can no longer buy the AR-55 or its ammo
/:cl:

### CODE REVIEW

**Comment by Pariah919** [comment]
You're hitting it with a triple nerf, one to the availability, then to NV (Which is fair honestly) then to aim mode? I think that's quite a lot considering its a 600~ req point weapon now in your PR rather than a SL weapon that's simply available in req. You should really decide on one or the other, removing it from SL vendor alongside this just sends it back into req/ert hell, at that point I'd just make it an ERT-Only weapon again.

HE mags are w/e, Lumi added them.

**Comment by yyzsong** [comment]
> Also Razor spam got removed too lel.

Guh, didn't notice.

**Comment by veterath** [comment]
removing HE and the scope is plenty enough, killing aimmode and roundstart functionality is absolutely overboard (roundstart price COULD increase, but considering it's about 25 points, it's really pushing it).

**Comment by yyzsong** [comment]
> You're hitting it with a triple nerf, one to the availability, then to NV (Which is fair honestly) then to aim mode? I think that's quite a lot considering its a 600~ req point weapon now in your PR rather than a SL weapon that's simply available in req. You should really decide on one or the other, removing it from SL vendor alongside this just sends it back into req/ert hell, at that point I'd just make it an ERT-Only weapon again.
> 
> Also Razor spam got removed too lel.
> 
> HE mags are w/e, Lumi added them.

I'm not really willing to budge on the aim mode removal, given that half of the pain of playing against it is having it shot at you through an advancing unga horde. It's still the exact same rifle in every other way, so you can get the same effects just with a bit more risk.

I've fiddled with the price a bit. Gl-54 is 150 points (You get two of them free anyways), and T-55 is down to 300 points. Decently cheap, but probably still limited based on the fact that people seem to be afraid of actually buying req weapons

**Comment by Pariah919** [comment]
> I've fiddled with the price a bit. Gl-54 is 150 points (You get two of them free anyways), and T-55 is down to 300 points. Decently cheap, but probably still limited based on the fact that people seem to be afraid of actually buying req weapons

Removing aimmode and NV from the scope just makes it a wacky bad T-70 honestly, if you ask me the issue with the gun is HE rounds (Which are impact grenades under a different meme) and Tanglefoot (0 second prime tanglefoot outside of artillery/mounted guns which are cumbersome and difficult to use in CQC is a big no no to me, just make it fire reskinned AGLS grenades.)

**Comment by yyzsong** [comment]
> Tanglefoot (0 second prime tanglefoot outside of artillery/mounted guns which are cumbersome and difficult to use in CQC is a big no no to me, just make it fire reskinned AGLS grenades.)

Honestly, I don't see much of a difference between instant det and delayed tanglefoot grenades, outside of the scenario where you have a warlock or shrike nearby the parry them and even if they manage to land one (doesn't seem to happen very often but it's not like we have data), the tfoot will still go off and deny some area. If aim mode were to stay and tangle was to be delayed, you'd still get the exact same thing happening but with one out of every so and so shots being reflected, or xenos being able to run a couple steps away before the gas hits them. It's the exact same thing you'd experience if someone just decided to roll with a GL-70 and tanglefoot nades.

> Req hell is optional, but removing aim mode is absolutely just too much combined with the other nerfs.

I'm not sure how removing aim mode castrates this gun in any way; it still retains the stats it had before, but just needs you to get a bit closer to the xenos to use it. Unless aim mode is legitimately that important to how the gun functions or distinguishes itself from the GL-70 (in which case I'd just try to make it ERT only), I think removing it is ok.

**Comment by yyzsong** [comment]
Why is oranges here now.

**Comment by yyzsong** [comment]
> If you want to remove its suppressive ability with HE/tangle

I mean tangle is still in. MJP can't edit my pr



> literal shotguns

I'd argue its grenades are definitely shotgun-ish. The standard airburst has okay damage, and can be used for incend as well. Maybe a slowdown reduction or just making the rifle better is in order if I wanna move this gun further to the front as an AR alternative but w/e.

**Review by Lumipharon — CHANGES_REQUESTED** [review]
The TX-55 is not supposed to be a good rifle, its a GL with a rifle attachment. So I'm not interested in making it perform better as a rifle.
Its a support platform specifically. The grenades are not comparable to normal grenades. Tfoot is a very small, directional cone vs the volume of a normal tfoot nade. HE is an extremely small explosion and also I believe the most expensive ammo in the game to use.

Lastly this PR nerfs 2 guns, not just one.

I'd rather just remove the TX-55 from the SL vendor than nerf it. I don't particular care if SL's get gamer gear.

**Comment by lKiseki** [comment]
> The TX-55 is not supposed to be a good rifle, its a GL with a rifle attachment. So I'm not interested in making it perform better as a rifle. Its a support platform specifically. The grenades are not comparable to normal grenades. Tfoot is a very small, directional cone vs the volume of a normal tfoot nade. HE is an extremely small explosion and also I believe the most expensive ammo in the game to use.
> 
> Lastly this PR nerfs 2 guns, not just one.
> 
> I'd rather just remove the TX-55 from the SL vendor than nerf it. I don't particular care if SL's get gamer gear.

That's not going to solve the issue of the AR-55 being insanely good. At this rate it'll be the next broken req weapon when the X-Ray rifle falls from grace.

**Comment by yyzsong** [comment]
> The TX-55 is not supposed to be a good rifle, its a GL with a rifle attachment. So I'm not interested in making it perform better as a rifle. Its a support platform specifically. The grenades are not comparable to normal grenades. Tfoot is a very small, directional cone vs the volume of a normal tfoot nade. HE is an extremely small explosion and also I believe the most expensive ammo in the game to use.
> 
> Lastly this PR nerfs 2 guns, not just one.
> 
> I'd rather just remove the TX-55 from the SL vendor than nerf it. I don't particular care if SL's get gamer gear.

As good as I'm gonna get, probably, and if I have to read the word commit one more time I'm gonna explode.



> That's not going to solve the issue of the AR-55 being insanely good. At this rate it'll be the next broken req weapon when the X-Ray rifle falls from grace. The sheer idiocy of a GL-54 underbarrel with no discernable drawback; what are basically rockets in an underbarrel attachment (HE mags); and the fact that it can aim mode all of this is just plain silly and needs to go.

argue with the maints for me thanks 🙏. You're stronger than I am.

**Comment by GrayRachnid** [comment]
A few things to note:
- Aim mode is tied to the scope it comes with.(meaning a T-47 makes it prone to FF).
- The scope it comes with gives a 0.3 slowdown (which makes it the support weapon it is). The NVG is just 5 (eyeballed)tiles and not the full distance. The scope is a mini-scope.
- HE mags on it are extremely rare, and expensive. I'm probably the one who's been using HE mags on it the most and I STILL prefer Airburst/Cloak/Tfoot. Incin mags are honestly awful and do more harm than help unless used strategically.
- The gun itself is not overpowered. Airburst is just very strong strategically and as a support weapon. On par with the AGL but with more expensive ammo.

Using it as a frontline weapon is akin to using a UGL but with more ammo. Something that makes sense considering a UGL costs nothing and this one is limited/req.

In short, the AR55 being 2-guns-in-one doesn't make it an overpowered gun, it just makes it a good and compact one. SLs(the people who get this in their vendor) are the ones who get to use that the most because they then don't have to juggle two guns to be both able to support /and/ defend themselves.

**Comment by FosterGrime-Heymynameslouie** [comment]
> Why is oranges here now.

yaeh did everyone just ignore that what

**Comment by GrayRachnid** [comment]
Title should be updated

**Review by Lumipharon — APPROVED** [review]
Goodnight my sweet prince.
